src/foam/nanos/notification/email/DAOEmailService.js

```diff
@@ -57,90 +52,18 @@ foam.CLASS({
                 return false;
               }
             };
-
-            protected EnvironmentConfiguration config_ = null;
-`
+          `
         }));
       }
     }
   ],
 
   methods: [
-    {
-      name: 'getConfig',
-      javaType: 'EnvironmentConfiguration',
-      args: [
-        {
-          name: 'group',
-          type: 'String'
-        }
-      ],
-      javaCode:
-`if ( config_ == null ) {
-  config_ = EnvironmentConfigurationBuilder
-    .configuration()
-    .resources()
-      .resourceLoaders()
-        .add(new TypedResourceLoader("dao", new DAOResourceLoader(getX(), group)))
-      .and()
-    .and()
-  .build();
-}
-return config_;`
-    },
     {
       name: 'sendEmail',
       javaCode: `
         getDao().inX(x).put(emailMessage);
```

Comment:
Do we need this implementation at all anymore or can we delete this entire file?

Reply:
Deleting the file. Thx for the catch